components/adblock_rust_ffi/src/lib.rs

```diff
@@ -373,3 +373,20 @@ pub unsafe extern "C" fn engine_hidden_class_id_selectors(
         .expect("Error: CString::new()")
         .into_raw()
 }
+
+#[cfg(feature = "ios")]
+#[no_mangle]
+pub unsafe extern "C" fn convert_rules_to_content_blocking(rules: *const c_char) -> *mut c_char {
+    let rules = CStr::from_ptr(rules).to_str().unwrap_or_else(|_| {
+        eprintln!("Failed to parse filter list with invalid UTF-8 content");
+        ""
+    });
+    let mut filter_set = adblock::lists::FilterSet::new(true);
+    filter_set.add_filter_list(&rules, Default::default());
+    // `unwrap` is safe here because `into_content_blocking` only panics if the `FilterSet` was not
+    // created in debug mode
+    let (cb_rules, _) = filter_set.into_content_blocking().unwrap();
```

Comment:
It seems odd to me to have FilterSet know how to convert itself to ios content blocking rules. I think it would make more sense to have something like `adblock::lists::ContentRuleList::FromFilterSet(filter_set)`, wdyt?

Reply:
This API has already been in `adblock-rust` for a while and I believe others are using it, I don't see a compelling reason to make a breaking change like that at the moment.

It's only available with an off-by-default feature flag, fwiw